.github/workflows/backend_lint_and_test.yml

```diff
@@ -59,8 +59,3 @@ jobs:
       # We are using severity level 'info' here.
       - name: Run dotnet format
         run: dotnet format --severity info --verbosity diagnostic --verify-no-changes --exclude ./api/migrations
-
-      - name: Run csharpier format
-        run: |
-          dotnet tool restore
-          dotnet csharpier --check
```

Comment:
Why is this removed?

Reply:
It has been failing in the last PRs so we decided to remove it and create an issue to be reintroduced later

Reply:
We can now rebase from main and re-add it since it has been fixed